src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2110,7 +2110,6 @@ private static ParameterSymbol GetParameter(int argIndex, MemberAnalysisResult r
             }
 
             // Otherwise, prefer methods with 'val' parameters over 'in' parameters and over 'ref' parameters when the argument is an interpolated string handler.
-            // PROTOTYPE(interp-string): Document in the spec
```

Comment:
Is there a corresponding PR for spec changes?

Reply:
Merged some time ago.